packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -3,9 +3,9 @@
 // found in the LICENSE file.
 
 import 'dart:math' as math;
-import 'dart:ui' as ui;
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
```

Comment:
travis is complaining about this line. You should just specify what exactly you're depending on in the next batch of imports.